exec.go

```diff
@@ -210,5 +212,20 @@ func execute(params execParams) error {
 		return err
 	}
 
+	if !params.detach {
+		p, err := os.FindProcess(process.Pid)
+		if err != nil {
+			return err
+		}
+
+		ps, err := p.Wait()
```

Comment:
What do you mean ? No, exec is not broken, we just never implemented the behavior related to detach == false.